state/factory/statedb.go

```diff
@@ -455,6 +478,9 @@ func (sdb *stateDB) State(s interface{}, opts ...protocol.StateOption) (uint64,
 	}
 	sdb.mutex.RLock()
 	defer sdb.mutex.RUnlock()
+	if cfg.Keys != nil {
+		return 0, errors.Wrap(ErrNotSupported, "Read state with keys option has not been implemented yet")
```

Comment:
add a comment like
// no Keys set, default to get all key-values in the namespace